lang/LangSource/SC_LanguageConfig.cpp

```diff
@@ -189,6 +189,10 @@ bool SC_LanguageConfig::writeLibraryConfigYAML(const Path& fileName)
 	out << Value << gPostInlineWarnings;
 
 	out << EndMap;
+	
+	bfs::path dir(fileName.parent_path());
+	if(!(bfs::exists(dir)))
+		postfl("WARNING: Cannot create config file. First create the directory '%s'\n", dir.c_str());
 
```

Comment:
IIUC, you just add here:

`return errFailed;`

Reply:
the check needs to go in the primitive (this is a routine called during the store primitive) but yes

Reply:
maybe the easiest is to first merge this and then add that.